src/components/legend/style.js

```diff
@@ -172,13 +172,14 @@ function styleBars(d) {
     barpath.exit().remove();
     barpath.each(function(d) {
         var w = (d.mlw + 1 || markerLine.width + 1) - 1,
```

Comment:
does this need `d` -> `d0` too? care to include line width and color arrays into the same mock?